### ISSUE TITLE
Liquibase doesn't fill a contexts column in a changelog table

### ISSUE THREAD

**Issue by misterflud** [liquibase/liquibase#1243]
## Environment
**Liquibase Version**:
3.5.5
**Liquibase Integration & Version**: <Pick one: CLI, maven, gradle, spring boot, servlet, etc.>
spring boot
**Liquibase Extension(s) & Version**: 

**Database Vendor & Version**:
Postgres 9.6
**Operating System Type & Version**:
any

## Description
I added this questions on stackOverFlow month ago \[[https://stackoverflow.com/questions/61784930/liquibase-doesnt-fill-contexts-column-in-changelog-table](https://stackoverflow.com/questions/61784930/liquibase-doesnt-fill-contexts-column-in-changelog-table)|url]
but I would like to repeate it here (I've tried on 3.8.0 and 3.5.5 liquibase in different Spring Boot):

My tree of changelog files is like this(for a prod, for a test):

```java
changelog-master.xml
--changelog-test.xml
----changelog-1.0.0
----changelog-1.1.0
--changelog-prod.xml
----changelog-1.0.0
----changelog-1.1.0
```

I try to use contexts and I find strange behaviour – if I use a context in a changelog-master.xml or a changelog-test.xml like this (parents changelog files):
`<include file="test/changelog-test.xml" relativeToChangelogFile="true" context="!prod"/>`

then liquibase doesn't fill a context column in a changelog table and a context column is null. But if I add an attribute 'context' to 'changeSet'
`<changeSet id="changelog_in_test_section" author="author@mail.ru" logicalFilePath="changelog-1.0.0" context="test">`
After that the context column is filled with '!prod AND test'
I want that the context column is alwayes filled (if I use a context attribute in parent changelog).

**Comment by molivasdat** [comment]
Hi @misterflud .  Thanks for taking the time to write this issue up.  We will add it to our list of issues.

**Comment by FBurguer** [comment]
Looks like it got fixed on `4.14`. If you still have the same issue, please try re-opening the issue. Thanks!
